packages/cxl-lumo-styles/scss/themes/vaadin-button.scss

```diff
@@ -21,3 +21,29 @@
 :host([theme="linkedin"]:hover)::before {
   opacity: 0.05;
 }
+
```

Comment:
I suggest applying common styles (border radius, size) to all buttons, not just these with `theme`.

Reply:
I did at first, but then got instructions to go for "primary" and "secondary" only.